packages/components/src/components/hds/application-state/header.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import TemplateOnlyComponent from '@ember/component/template-only';
+
+export interface HdsApplicationStateHeaderSignature {
+  Args: {
+    title?: string;
```

Comment:
**Question:** Are you sure this is optional? Could you check that? I think this _might_ provide the ACCNAME in which case, it shouldn't be optional.

Reply:
Fair question – I made that comment based on the current documentation. Happy for us to revisit this.

Reply:
If it's according to the docs, then it's fine; but we should probably re-visit this component and look at the pattern/guidance and see if everything really should be optional.

Reply:
I created [a task](https://hashicorp.atlassian.net/browse/HDS-3384) for us to look into it.

Reply:
Thanks for the discussion all, and for creating that task @alex-ju